src/overview/bare-metal.md

```diff
@@ -1,59 +1,57 @@
-# Bare Metal (`no_std`)
+# Developing on Bare Metal (`no_std`)
 
-Using `no_std` may be more familiar to embedded Rust developers; it does not use `std` (the Rust standard library) but instead uses a subset, the `core` library. The [official Rust embedded book] has a [great section on this].
+Using `no_std` may be more familiar to embedded Rust developers; it does not use `std` (the Rust [`standard`][rust-lib-std] library) but instead uses a subset, the [`core`][rust-lib-core] library. [The Embedded Rust Book][embedded-rust-book] has a great [section][embedded-rust-book-no-std] on this.
 
-It's important to note that in general a `no_std` crate can always compile in `std` environment but the inverse is not true. Therefore, when creating crates it's worth keeping in mind if it needs the standard library to function.
+It's important to note that since `no_std`uses the Rust core library, a subset of the Rust standard library,  a `no_std` crate can compile in `std` environment but the oposite is not true. Therefore, when creating crates it's worth keeping in mind if it needs the standard library to function.
```

Comment:
```suggestion
It's important to note that since `no_std`uses the Rust `core` library, a subset of the Rust `standard` library,  a `no_std` crate can compile in `std` environment but the oposite is not true. Therefore, when creating crates it's worth keeping in mind if it needs the `standard` library to function.
```

Reply:
> `no_std`uses

MIssing space there

> Rust `standard` library`

I think it's better to be consistent and use either "Rust standard library" or `std`

Reply:
> I think it's better to be consistent and use either "Rust standard library" or std

This is something that should probably also be included in the guidelines (https://github.com/esp-rs/book/pull/79)

Reply:
Didn't even know that PR existed, will take a look